lib/src/typings.d.ts

```diff
@@ -16,8 +16,16 @@ declare namespace RPE {
   }
 
   interface ConfigureOptions {
-    promiseMiddleware: boolean,
-    toolkitMiddleware: boolean,
+    promise: boolean,
+    toolkit: boolean,
+    saga: boolean,
     ignoredActionTypes: string[]
   }
+
+  interface ConfigureOutput<T, K> {
+    middlewares: T[],
+    sagaOptions?: {
```

Comment:
make this prop required
always return sagaOptions object with effectMiddlewares default empty array
if user enables saga, then return our saga middleware inside array